static/app/components/events/interfaces/threads/threadSelector/index.tsx

```diff
@@ -50,28 +47,76 @@ function getThreadLabel(
 
 function ThreadSelector({threads, event, exception, activeThread, onChange}: Props) {
   const organization = useOrganization({allowNull: true});
+  const [currentThread, setCurrentThread] = useState<Thread>(activeThread);
+  const [sortAttribute, setSortAttribute] = useState<SortAttribute>(SortAttribute.ID);
+  const [isSortAscending, setIsSortAscending] = useState<boolean>(true);
 
   const hasThreadStates = threads.some(thread =>
     defined(getMappedThreadState(thread.state))
   );
+  const threadInfoMap = useMemo(() => {
+    return threads.reduce((acc, thread) => {
+      acc[thread.id] = filterThreadInfo(event, thread, exception);
+      return acc;
+    }, {});
```

Comment:
needs a type, currently implicit any
```suggestion
    return threads.reduce<Something>((acc, thread) => {
      acc[thread.id] = filterThreadInfo(event, thread, exception);
      return acc;
    }, {});
```

Reply:
good catch :+1: